services/director-v2/src/simcore_service_director_v2/api/errors/validation_error.py

```diff
@@ -0,0 +1,29 @@
+from typing import Union
+
+from fastapi.exceptions import RequestValidationError
+from fastapi.openapi.constants import REF_PREFIX
+from fastapi.openapi.utils import validation_error_response_definition
+from pydantic import ValidationError
+from starlette.requests import Request
+from starlette.responses import JSONResponse
+from starlette.status import HTTP_422_UNPROCESSABLE_ENTITY
+from fastapi.encoders import jsonable_encoder
+
+
+async def http422_error_handler(
+    _: Request,
+    exc: Union[RequestValidationError, ValidationError],
```

Comment:
it did not work with raising 500s from ValidationError?

Reply:
that is another handler. Check the application init and see that there is a special handler for ``HTTPError``